targets/targets.json

```diff
@@ -2356,11 +2356,12 @@
         "core": "Cortex-A9",
         "program_cycle_s": 2,
         "extra_labels": ["RENESAS", "MBRZA1H"],
+        "macros": ["TARGET_FEATURE_EXTENSION_REGISTER_COUNT_GCC=32", "TARGET_FEATURE_EXTENSION_REGISTER_COUNT_IAR=32"],
```

Comment:
What does these 2 macro do? Would it be better to put into the config, instead of macro ?

I can see it is used in  the rtx assembly files, are these also part of RTX ?

Reply:
Original was the form of `TARGET_FEATURE_EXTENSION_REGISTER_COUNT`.
However, since this macro is a built-in environment variable of the ARM compiler,
in the case of GCC or IAR, this macro is became un-active, either process related to VFP does not run.
The macro here should be the target dependent value. In order to run the processing with the register count 32 as same previously, 
I prepared the renamed macro, I actived it's processes in GCC and IAR.
Finally there was no a file is included in RTX assembly file, so I defined a macro in `targets.json`.

Reply:
Can we do this via config, see an example: https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L269. Rather than macro with some value?

Reply:
As a result of cherry-picking fixes from official CMSIS repo please, the macro that I added in targets.json became unnecessary. So I reverted the change of its file.